projects/packages/forms/src/dashboard/hooks/use-inbox-data.ts

```diff
@@ -116,13 +116,18 @@ export default function useInboxData(): UseInboxDataReturn {
 		}
 	);
 
+	const isLoadingData =
+		( ! rawRecords?.length && ! hasResolved ) ||
+		isLoadingInboxData ||
+		isLoadingSpamData ||
+		isLoadingTrashData;
+
 	return {
 		totalItemsInbox,
 		totalItemsSpam,
 		totalItemsTrash,
 		records,
-		isLoadingData:
-			isLoadingRecordsData || isLoadingInboxData || isLoadingSpamData || isLoadingTrashData,
+		isLoadingData,
```

Comment:
Is there anything I need to do to see/duplicate the 'flash' issue? I'd like to confirm the issue before testing the fix, but I don't typically see a flash when the form responses page loads.